### PR TITLE
New Installation Docs

### PR DESCRIPTION
**Pull request by robmoggach**
Updated, cleaned up installation docs...

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -1,111 +1,135 @@
...
 ```bash
 cd freqtrade
 ```
-4. Copy `config.sample` to `config.json`
+
+#### 1.4. Copy `config.json.example` to `config.json`
+
 ```bash
-cp config.json.example config.json
+cat config.json.example >> config.json
```
Comment (shusso) [inline docs/installation.md]: We are copying a file, so I'd rather stick to `cp` rather than `cat`

Comment (robmoggach) [reply docs/installation.md]: `cp` will clobber any existing file... `cat >>` will append and not overwrite... but I see both points...

Comment (shusso) [reply docs/installation.md]: will append yes, but then what are the likelyhoods that the appended config file is valid json anymore?

Comment (glonlas) [reply docs/installation.md]: in this case I suggest to add the param -n

`cp -n config.json.example config.json`

**Comment by robmoggach** [comment]
changed `cat` to `cp -n`

**Comment by glonlas** [comment]
Hi @robmoggach, could you please rebase from develop and solve the conflict, then we are good to go.

**Comment by robmoggach** [comment]
haven't done that before - explain and i'll do it... thanks

--
Robert Moggach
http://moggach.com
Mobile (647) 544-1793

**Comment by glonlas** [comment]
Because you did your update directly on your develop branch, I do not know what will be the result. 
For next updates, I suggest to always use a branch.

1. `git checkout develop` (switch back to develop branch)
2. `git pull` (get the last update from the official develop branch)
3. `git checkout -b my_branch_name` (create a new branch from the content of develop)
4. do your updates
5. `git push origin my_branch_name` (push your branch to your repo)

then when develop got updated, you just have to git pull develop to get the latest updates, then rebase from develop.

6. `git checkout develop` (switch back to develop branch)
7. `git pull` (get the last update from the official develop branch)
8. `git checkout my_branch_name` (go back to your branch)
9. `git rebase develop` (integrate the last updates from develop into your branch)
10. `git push origin my_branch_name`

**Comment by glonlas** [comment]
I will check how to remove your conflicts

**Pull request merged**